lua/telescope/pickers.lua

```diff
@@ -77,6 +77,7 @@ function Picker:new(opts)
     entry_prefix = vim.F.if_nil(opts.entry_prefix, config.values.entry_prefix),
     multi_icon = vim.F.if_nil(opts.multi_icon, config.values.multi_icon),
 
+    theme = vim.F.if_nil(opts.theme, "dropdown"),
```

Comment:
doesn't this change the default theme?

Reply:
oh my bad, I must have though dropdown was the default when I wrote this! Is the default one just unnamed?